src/client/components/ui/Meilisearch/MeilisearchCustomCurrentRefinements.tsx

```diff
@@ -19,11 +20,13 @@ export default function MeilisearchCustomCurrentRefinements(props: UseCurrentRef
 		<ul aria-label="liste des filtres appliqués" className={styles.tagList}>
 			{items.map((item) => {
 				return item.refinements.map((refinement, index) => (
-					<li key={index} className={styles.tag}>
-						<span>{getCapitalizedItems(refinement.label)}</span>
-						<button aria-label="supprimer le filtre" type="button" onClick={() => refine(refinement)}>
-							<Icon name="close" />
-						</button>
+					<li key={index} >
+						<Tag>
+							<button aria-label="supprimer le filtre" type="button" onClick={() => refine(refinement)}>
+								{getCapitalizedItems(refinement.label)}
+								<Icon name="close" />
+							</button>
```

Comment:
Juste un petit souci avec l'`aria-label` ->
Comme il est sur le bouton, il va prendre le dessus sur le contenu qui est à l'intérieur. Ce qui sera vocalisé sera uniquement "supprimer le filtre" et le label ne sera pas vocalisé. 
On pourrait reprendre le `refinement.label` dans l'aria-label et avoir par exemple : 
```suggestion
							<button aria-label={`${refinement.label} - supprimer le filtre`} type="button" onClick={() => refine(refinement)}>
								{getCapitalizedItems(refinement.label)}
								<Icon name="close" />
							</button>
```


Pour plus d'info sur la manière dont son calculé les noms accessibles, j'ai été fouiller sur : 
https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8#:~:text=Per%20the%20Accessible%20Name%20and,text%20supplied%20within%20the%20link. (celui ci concerne le lien mais est assez concis) 
https://www.w3.org/TR/accname/#terminology (celui ci est complet mais assez indigeste 😄 )